apps/trading/components/footer/footer.tsx

```diff
@@ -17,7 +17,8 @@ export const Footer = () => {
     </footer>
   );
 };
-
+const MAINNET_INCIDENT_LINK =
```

Comment:
chore (blocking): we'll probably want to move this to env var

Reply:
done as well